src/style.css

```diff
@@ -0,0 +1,69 @@
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.conteiner {
+  position: relative;
+}
+
+.stars {
+  display: flex;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  border-radius: 0.5px;
+  margin-right: 4px;
```

Comment:
the last star shouldn't have margin-right: 4px

Reply:
You can remove margin-right here because you have it below